LICENSE

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Please don't edit the license file — we could keep it exactly like https://github.com/google/new-project/blob/master/LICENSE.

Reply:
Ok

Reply:
This has been done in #272.

Reply:
I think both PRs are ready to be merged now.